ocfweb/urls.py

```diff
@@ -28,6 +29,7 @@
 
     url(r'^$', home, name='home'),
     url(r'^robots\.txt$', robots_dot_txt, name='robots.txt'),
+    url(r'^sign\.json$', sign_text, name='sign.json'),
```

Comment:
+1, /api/ sounds great. i was going to comment that we should move it out of the root, but couldn't think of a good name. /api/ is great.